frontend/src/pages/Assignments/AssignmentRubric/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { useContext, useEffect, useState } from "react";
```

Comment:
Once a new rubric list has been made, the user should be redirected back into the rubric page as a "confirmation" of it being created. Right now I have to click cancel and then import the rubric manually and IMO this should already happen behind the scenes if a user saves a new rubric